### DISCUSSION TITLE
windows based images, please?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [microsoft/openjdk#203] [Q&A] [answered]
Microsoft provides Linux based containers, but does not support its own Windows containers - **nano** and **core**. Any plans to fix  that?

**Comment by brunoborges** [comment, marked as answer]
@hrstoyanov we haven't seen customer demand yet.

Could you share a bit of your use case?

**Reply by hrstoyanov** [reply]
@brunoborges - developing an intelligent ms sql proxy . I can use Linux container, but if my customer wants to deploy on premise, where they do not have any linux, I was hoping to be able to package my proxy as a windows native container.

On the fqce of it, this should be trivial to build. I am not sure if the JVM would require NANO or CORE . I see people doing it, but official image from Microsoft would be greatly appreciated.

**Reply by brunoborges** [reply]
If the customer is deploying a Windows container to Azure, we will provide commercial support regardless. Support wouldn't be available to those deploying on-premises.

**Reply by hrstoyanov** [reply]
No, the customer should not be forced to deploy on Azure, and yes, on-premise is the problem. I hate to make mu customer use Linux, if they are 100% Windows shop.

**Reply by brunoborges** [reply]
They are more than welcome to grab the MS Build of OpenJDK and build a Windows container image for their purpose!

The license is GPL2+CE as usual, and they can benefit from the guaranteed public updates we will push to LTS versions.
